src/main/java/org/jenkinsci/test/acceptance/junit/WithCredentials.java

```diff
@@ -43,6 +44,7 @@
 @Target({METHOD, TYPE})
 @Inherited
 @Documented
+@WithPlugins("credentials@2.0.7")
```

Comment:
That's what I thought too; but I threw the annotation on there and tried running a test without credentials included in it's list of plugins, and it installed 2.0.7 for that test.  It feels a bit off using that annotation on another annotation, but I don't expect people writing new tests using the WithCredentials rule to know to add the credentials plugin in their list of required plugins.